prometheus/promhttp/http.go

```diff
@@ -371,6 +376,8 @@ type HandlerOpts struct {
 	// (which changes the identity of the resulting series on the Prometheus
 	// server).
 	EnableOpenMetrics bool
+	// TODO: document this properly
+	EnableOpenMetricsCreatedLines bool
```

Comment:
```suggestion
	EnableOpenMetricsCreatedMetrics bool
```

Nice, let's do extensive docs here on how to use it and what are the consequences.

Reply:
Also I still think this is dangerous option due to name conflicts and size of scrape 🙈

Reply:
> Also I still think this is dangerous option due to name conflicts and size of scrape 🙈

Not sure what can we do about that besides documenting it properly 😬